gatsby/content/blog/2022/12/2022-12-07-synapse-1.73.mdx

```diff
@@ -0,0 +1,57 @@
+---
+date: '2022-12-07'
+title: Synapse 1.73 released
+categories:
+- Releases
+author: Mathieu Velten
+---
+
+And here is another update to your beloved Matrix server implementation, [Synapse
+1.73](https://github.com/matrix-org/synapse/releases/tag/v1.73.0).
+
+## Announcements
+
+### Legacy Prometheus metric names removed
+
+As mentioned in previous blogpost, after disabling comes removing !
+
+`enable_legacy_metrics` has been removed, sorry if it breaks your dashboard but
+the upgrade should not be that difficult, and you got some heads-up.
+
+You can still find the full list of renamed metrics
+[here](https://matrix-org.github.io/synapse/v1.73/metrics-howto.html#renaming-of-metrics--deprecation-of-old-names-in-12).
+
+## The new stuff
+
+### Performance
+
+A bunch of performance improvements have been included in this release.
+You can expect `/messages` to be [quite faster](https://github.com/matrix-org/synapse/pull/14527).
+
+Various optimizations have also been done on fetching aggregations of relationships.
```

Comment:
We can link the PRs:

 - https://github.com/matrix-org/synapse/pull/14491
 - https://github.com/matrix-org/synapse/pull/14508
 - https://github.com/matrix-org/synapse/pull/14510

And add some graphs from: https://github.com/matrix-org/synapse/issues/13624#issuecomment-1323948527